setup.py

```diff
@@ -151,6 +157,7 @@ def _set_builtin(name, value):
     "cmdclass": cmdclass,
     "ext_modules": [udunits_ext],
     "package_data": get_package_data(),
+    "options": {"bdist_wheel": {"py_limited_api": "cp311"}},
```

Comment:
Assuming `cp311` is liable to move forward, there should be a line in `test_python_versions()`:

https://github.com/SciTools/cf-units/blob/9e34057547f407df84fc3d5bc742f93831cbbe15/cf_units/tests/test_coding_standards.py#L83-L159